rpm/SOURCES/couchdb-nouveau.service

```diff
@@ -4,8 +4,8 @@ Wants=network-online.target
 After=network-online.target
 
 [Service]
-Environment="JAVA_OPTS=-server"
-ExecStart=/opt/couchdb/nouveau/bin/nouveau server /opt/couchdb/etc/nouveau.yaml
+EnvironmentFile=/etc/sysconfig/couchdb-nouveau
+ExecStart=/bin/sh -c "exec /usr/bin/java ${JAVA_OPTS} -jar /opt/couchdb/nouveau/lib/nouveau-*.jar server /opt/couchdb/etc/nouveau.yaml"
```

Comment:
the only reason the .deb has /bin/sh is because the jvm home was configurable (systemd insists that the first item in ExecStart is an absolute path and doesn't consider variable substitution so `ExecStart=${JAVA_HOME}` was prohibited. 

If the rpm version won't allow jvm location customisation (why not btw?) then just do `ExecStart=/usr/bin/java ...`

Reply:
change`/usr/bin/java` to `${JAVA_HOME}/bin/java`?